Makefile

```diff
@@ -0,0 +1,37 @@
+PWD := $(shell dirname $(realpath $(lastword $(MAKEFILE_LIST))))
+
+GOPKG = github.com/bndw/pick
+GOPATH = "$(CURDIR):$(CURDIR)/vendor"
+
+BIN_DIR = /usr/local/bin
+INSTALL = install
+
+FOLDERS = $(shell find . -mindepth 1 -maxdepth 1 -type d -not -path "*.git" -not -path "*vendor" -not -path "*bin")
+
+all: build
+
+goget:
+	GOPATH=$(GOPATH) go get github.com/rogpeppe/godeps
```

Comment:
Why not something [recommended](https://github.com/golang/go/wiki/PackageManagementTools) or standard like https://github.com/tools/godep?